### DISCUSSION TITLE
Error: ingesting to DCC registry is forbidden

### DISCUSSION THREAD

**Discussion by marisalim** [nih-cfde/published-documentation#151] [Q&A] [answered]
```
# on checking status after cfde-submit run
This Flow has completed.
Your CFDE submission () failed to ingest into DERIVA with this error:
Error ingesting to DERIVA: Submission to DCC cfde_registry_dcc:hmp is forbidden
```

**Comment by ACharbonneau** [comment, marked as answer]
This error You have used a valid DCC ID, but you are not listed as a Submitter for that DCC

**Possible solutions:**

1. Get the correct DCC ID for your institution

You may have a simple copy/paste error where you got the wrong ID from the ID list. Follow the [instructions here](https://docs.nih-cfde.org/en/latest/cfde-submit/docs/#find-your-dccs-identifier) to find the correct identifier

2. Get Submitter credentials

If you have the correct identifier for your institution, then you may not have the correct credentials. A Principle Investigator or Program Manager at your DCC will have [given credential information to the CFDE](https://github.com/nih-cfde/published-documentation/wiki/Onboarding-to-the-CFDE-Portal-Submission-System). Your contact should email [the helpdesk](support@cfde.atlassian.net) with updated information.
